src/minimega/vm.go

```diff
@@ -512,3 +512,50 @@ func ParseVmState(s string) (VmState, error) {
 
 	return VM_ERROR, fmt.Errorf("invalid state: %v", s)
 }
+
+// Get the VM info from all hosts optionally applying column/row filters.
+// Returns a map with keys for the hostnames and values as the tabular data
+// from the host.
+func globalVmInfo(masks []string, filters []string) map[string][][]string {
```

Comment:
We're no longer doing this right? Instead we're just letting vm info pack the vms.vms (or whatever it needs to be) into the Data element of the minicli response?